tests/Integration/PreventEmptySessionCookieIntegrationTest.php

```
@@ -0,0 +1,285 @@
+
+        $sessionId = session_id();
+        self::assertNotFalse($sessionId, 'Session ID should be generated');
+
+        session_write_close();
+        PreventEmptySessionCookie::checkAndCleanup();
+
+        $dataInRedis = $this->connection->get($sessionId);
+        self::assertFalse($dataInRedis, 'Empty session should not be written to Redis');
+
+        self::assertSame(PHP_SESSION_NONE, session_status(), 'Session should be destroyed after cleanup');
+    }
```

PHPMD (2.15.0)

testEmptySessionDoesNotWriteToRedisAndDeletesCookie accesses the super-global variable $_COOKIE. (undefined)

(Superglobals)